doc/user-guide/cli.md

```diff
@@ -178,6 +178,33 @@ The deployment process started! Track its status at: https://www.platform.ploomb
 
 Here, `project-id` is `muddy-art-3373` and `job-id` is `8d1067be`.
 
+## Retrieve application logs
+
+You can retrieve app logs during or after deployment using the `logs` command. You must pass a `project-id` or `job-id`. Here's an example using `--project-id`:
```

Comment:
user might not now what a project-id or job-id is, show an example

btw, what happens if we have a custom name? I think the project-id won't be visible and passing the custom name as the project-id won't work, please check and add any notes here (and issues in the CLI repo if needed)

Reply:
1. [added example ids](https://github.com/ploomber/doc/pull/245/files#diff-bdba8f4b461f494edc3b95709bb0d5d869d0d6d1593c34c1ac790c9f24787e64R185-R195)
2. you're right, I just tested it and it doesn't work by passing a custom name. I created an issue to fix this here: https://github.com/ploomber/cli/issues/126. @edublancas please take a look, it might be a good idea to generalize this fix since the same issue will apply to any command that accepts `project-id`